src/app/shared/navbar/searchbar/searchbar.ts

```diff
@@ -0,0 +1,51 @@
+import {Component, HostBinding} from '@angular/core';
+import {Router} from '@angular/router';
+import {FormControl} from '@angular/forms';
+import {MdSnackBar} from '@angular/material';
+import {Observable} from 'rxjs';
```

Comment:
To elaborate: when you import from just `rxjs`, it brings in a lot of than just what you're using and increases the payload size.